backend/sozisel/lib/sozisel_web/schema/subscriptions/topics.ex

```diff
@@ -10,4 +10,8 @@ defmodule SoziselWeb.Schema.Subscriptions.Topics do
   def session_all_participants(session_id) do
     "session:#{session_id}:participants"
   end
+
+  def poll(launched_event_id) do
+    "poll:#{launched_event_id}"
+  end
```

Comment:
Wy we do not use topic with name `session_all_participants`? It send to all participants in session.

Reply:
I guess I can change that but I don't think that it makes anything more clear than it is...

Reply:
I see one situation where having `session_all_participants` is wrong, when 2 consecutive polls come in then a single subscription will still be a valid one and will receive just different result but in fact it should be invalidated. Therefore I will keep the topic separate.